manager/src/main/java/com/redhat/service/bridge/manager/models/Bridge.java

```diff
@@ -10,8 +10,13 @@
 import javax.persistence.UniqueConstraint;
 
 @NamedQueries({
-        @NamedQuery(name = "BRIDGE.findByStatusesAndShardId",
-                query = "from Bridge where status IN :statuses and shard_id=:shardId"),
+        @NamedQuery(name = "BRIDGE.findByShardIdWithReadyDependencies",
```

Comment:
`findByShardIdWithReadyDependencies` the name is a bit misleading since for instances in DEPROVISION we check if the dependencyStatus is DELETED

Reply:
See https://issues.redhat.com/browse/MGDOBR-456